src/Tools/Metrics/MetricsOutputWriter.cs

```diff
@@ -61,7 +61,7 @@ void writeHeader()
                 switch (data.Symbol.Kind)
                 {
                     case SymbolKind.NamedType:
-                        writer.WriteAttributeString("Name", data.Symbol.ToDisplayString(SymbolDisplayFormat.MinimallyQualifiedFormat));
```

Comment:
I don’t think we want to use fully qualified names for all type names. Also this format adds global:: prefix. Can we just qualify with parent type name for nested types?

Reply:
Ok! Yet I am not sure if it's not better to use the fully qualified name for nested types rather than concatenating with parent name. In case of multiple level of nesting, is it fine to simply display current level + parent? It seems to still be confusing so we will need to repeat the concat for all containing types.

WDYT?